src/core/gps.py

```diff
@@ -57,7 +57,7 @@ def _toDoInsideLoop(self):
             if self._gps:
                 self._gps.close()
                 self._gps = None
-            if not os.environ.get("SAMMO_DEBUG"):
+            if os.environ.get("SAMMO_DEBUG"):
                 print("GPS offline - exception when trying to read")
```

Comment:
We could probably use the logger instead of `print` in this case and this way hide the `SAMMO_DEBUG` mechanism?

Reply:
Yes, it's true I even forgot we had a Logger class in sammo :sweat_smile: